.github/workflows/ci-rust.yml

```
@@ -0,0 +1,73 @@
+      contents: write
+
+    defaults:
+      run:
+        working-directory: rust
+
+    steps:
+      - name: Checkout Code
+        uses: actions/checkout@v4
+
+      - name: Install Rust
+        uses: actions-rs/toolchain@v1
```

actionlint (1.7.7)

the runner of "actions-rs/toolchain@v1" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)